4-storage/postgres-storage/sql/atomic-def.sql

```diff
@@ -151,7 +149,29 @@ CREATE TABLE "atomic"."events" (
 	"doc_charset" varchar(128),
 	"doc_width" integer,
 	"doc_height" integer,
-	PRIMARY KEY ("event_id")
+	-- Currency
+	"tr_currency" varchar(16),              -- Added in 0.4.0
+	"tr_total_base" decimal(18, 2),         -- Added in 0.4.0
+	"tr_tax_base" decimal(18, 2),           -- Added in 0.4.0
+	"tr_shipping_base" decimal(18, 2),      -- Added in 0.4.0
+	"ti_currency" varchar(16),              -- Added in 0.4.0
+	"ti_price_base" decimal(18, 2),         -- Added in 0.4.0
+	"base_currency" varchar(16),            -- Added in 0.4.0
+	-- Geolocation
+	"geo_timezone" varchar(64),             -- Added in 0.4.0
+	-- Click ID
+	"mkt_clickid" varchar(64),              -- Added in 0.4.0
+	"mkt_network" varchar(64),              -- Added in 0.4.0
+	-- ETL tags
+	"etl_tags" varchar(500),                -- Added in 0.4.0
+	-- Time event was sent
+	"dvce_sent_tstamp" timestamp,           -- Added in 0.4.0
+	-- Referer
+	"refr_domain_userid" varchar(36),       -- Added in 0.4.0
+	"refr_dvce_tstamp" timestamp,           -- Added in 0.4.0
+	-- Derived contexts
+	"derived_contexts" json                 -- Added in 0.4.0
+                                            -- Removed primary key constraint on event_id
```

Comment:
Can you align -- Removed with lines above, and make it "Removed in 0.4.0 primary ..."